utils/memcpy-bench/CMakeLists.txt

```diff
@@ -15,6 +15,10 @@ add_executable (memcpy-bench
 
 add_compile_options(memcpy-bench PRIVATE -fno-tree-loop-distribute-patterns)
 
+if (OS_SUNOS)
+	target_compile_options(memcpy-bench PRIVATE "-Wa,--divide")
```

Comment:
The implementation of the memcpy assembly uses a `/` character, which on illumos's assembler is interpreted as the start of a comment, for compatibility with older assemblers.